### DISCUSSION TITLE
[Feature]: A way to see starred/liked photos

### DISCUSSION THREAD

**Discussion by fivestones** [immich-app/immich#1640] [Feature Request]
### Feature detail

It would be great if there was an album that just showed photos that have been starred.

Photos that are liked (hearted) on an iPhone show up in immich after being backed up with a star on them, which is great--but I don't see any way yet to show just these photos. Maybe this already exists and I just haven't found it, but I didn't see it mentioned in the documentation or anywhere on github. Or maybe this is coming along with the yet-to-come search function?

Thanks!

### Platform

Web

**Comment by jrasm91** [comment]
I think there are a few of these "views". One for trashed, archived, favorited, marked as private, etc. 

It would be great if the fancy timeline worked for these collections as well, but that will require some refactoring in the web. Regardless, there is some work to do to have any easily reusable component for these pages.

**Comment by fivestones** [comment]
Maybe instead of just a set of a few views, there could be a filter to pick and choose what you want displayed in the timeline. Like, show me all photos that were taken in 2002 in Europe that are starred. Or show me all photos from San Diego that were NOT taken in 2018 that are landscape.

It would be nice to set filters for the search too. I.e., use the CLIP search on the one of the above filtered views.
